contracts/protocol/modules/v1/AuctionRebalanceModuleV1.sol

```diff
@@ -0,0 +1,1306 @@
+/*
+    Copyright 2023 Index Coop
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    SPDX-License-Identifier: Apache License, Version 2.0
+*/
+
+pragma solidity 0.6.10;
```

Comment:
Did you by any chance evaluate how much effort it would have been to implement this in 0.8 as well ?

I guess the effort would lie mostly in bumping all of the `../../lib` dependencies also to 0.8 but in some cases that might mean only bumping the pragma and removing SafeMath😅.

Reply:
Yep you're exactly right, bumping to 0.8 for the module would require bumping the lib dependencies, including `ModuleBase` and `Position`.